pom.xml

```
@@ -12,7 +12,7 @@
     <packaging>hpi</packaging>
     <version>${revision}${changelist}</version>
     <name>Text Finder</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/Text-finder+Plugin</url>
+    <url>https://github.com/jenkinsci/text-finder-plugin</url>
```

Can we avoid hard-coding the name of the plugin by relying on `artifactId` instead? That is how it is done in the archetype as well:

https://github.com/jenkinsci/archetypes/blob/8a49c0c784267d8baf3841199fa4292aad18225a/empty-plugin/src/main/resources/archetype-resources/pom.xml#L26

```suggestion
    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```